routes/login.js

```diff
@@ -1,21 +1,20 @@
 const express = require('express');
 const bcrypt = require('bcrypt');
 const User = require('../models/user.js');
-const jwt = require('jsonwebtoken');
+const generateToken = require('../tokens/generateToken');
 const asyncHandler = require('../middlewares/asyncMiddleware');
 
 const router = express.Router();
 
-const generateToken = (user) => {
-  const signature = 'my-super-secret';
-  const expiration = '6h';
-  return jwt.sign({ user }, signature, { expiresIn: expiration });
-};
-
 router.post('/', asyncHandler(async (req, res) => {
   const { username, password } = req.body;
-  try {
     const user = await User.findOne({ where: { username: username }});
+    if(user === null) {

```

Comment:
Missing space in formatting here, check can be simplified or even omitted using `rejectOnEmpty`.